controllers/templates.go

```diff
@@ -68,7 +68,7 @@ spec:
             exclude:
               - kube-*
             include:
-              - default
+              - openshift-cluster-group-upgrades
```

Comment:
you don't need to add this. THe namespace openshift-cluster-group-upgrades is a namespace where the controller runs, which is on the hub. This namespace selector field is on configuration policies, which are meant on the spokes.
As a matter of fact, this 'default' is a bug, I should probably change to * .